changelog/new_group_block.md

```diff
@@ -0,0 +1 @@
+* [#356](https://github.com/rubocop/rubocop-ast/pull/356): Add `Node#any_block_type?` to determine if a node is either a block or numblock. ([@earlopain][])
```

Comment:
I would start by saying that an alias `:any_block` was added for `block` and `numblock` and then mention the method